RELEASE-NOTES.md

```diff
@@ -14,6 +14,7 @@
 * Empty strings are now detected as invalid language codes in the term classes
 * Made the `Item` constructor parameters optional
 * Made the `Fingerprint` constructor parameters optional
+* Made the `Property` constructor parameter for fingerprint nullable
```

Comment:
Just fix it then. The README is documentation, and this never required RFCs to modify